### PR TITLE
Factorio: Fix typos in options tooltip

### PR DESCRIPTION
**Pull request by budak7273**
## What is this fixing or adding?

- Fix some typos in the Factorio options tooltip for Tech Tree Information
- Add more details to the tooltip for the Advancement option

## How was this tested?

Locally running WebHost.py after making the changes, used to capture the screenshot below

## If this makes graphical changes, please attach screenshots.

Before: 
image

After: 
image

### CODE REVIEW

**Review by Silvris — APPROVED** [review]
Looks good to me!

File: worlds/factorio/Options.py [hunk 1]
```diff
@@ -146,8 +146,8 @@ class TechTreeLayout(Choice):
 
 class TechTreeInformation(Choice):
     """How much information should be displayed in the tech tree.
-    None: No indication what a research unlocks
-    Advancement: Indicators which researches unlock items that are considered logical advancements
+    None: No indication of what a research unlocks
+    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for
```
Comment (ScipioWright) [inline worlds/factorio/Options.py:150]:
```suggestion
    None: No indication of what a research unlocks.
    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for.
```

To be more consistent with the "Full" option line.

**Review by ScipioWright — APPROVED** [review]
Very minor edit for consistency.

File: worlds/factorio/Options.py (hunk 1 above)
Comment (nicholassaylor) [inline worlds/factorio/Options.py:150]:
```suggestion
    Advancement: Indicates if a research unlocks an item that is considered logical advancement, but not who it is for
```
Small suggestion for readability

**Review by nicholassaylor — CHANGES_REQUESTED** [review]

File: worlds/factorio/Options.py (hunk 1 above)
Comment (budak7273) [reply worlds/factorio/Options.py]: Ah, by applying the other one I have made yours outdated, dang it.

File: worlds/factorio/Options.py [hunk 2]
```diff
@@ -146,8 +146,8 @@ class TechTreeLayout(Choice):
 
 class TechTreeInformation(Choice):
     """How much information should be displayed in the tech tree.
-    None: No indication what a research unlocks
-    Advancement: Indicators which researches unlock items that are considered logical advancements
+    None: No indication of what a research unlocks.
+    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for.
```
Comment (budak7273) [inline worlds/factorio/Options.py:150]:
```suggestion
    Advancement: Indicates if a research unlocks an item that is considered logical advancement, but not who it is for.
```

**Review by black-sliver — APPROVED** [review]

File: worlds/factorio/Options.py (hunk 1 above)
Comment (budak7273) [reply worlds/factorio/Options.py]: Handled in 7bc046f

**Pull request merged**